base/tuple.jl

```diff
@@ -142,6 +146,7 @@ ntuple(f, ::Type{Val{15}}) = (@_inline_meta; (f(1), f(2), f(3), f(4), f(5), f(6)
 
 function ntuple(f::F, ::Type{Val{N}}) where {F,N}
     Core.typeassert(N, Int)
+    (N >= 0) || throw(ArgumentError(string("tuple length should be ≥0, got ", N)))
```

Comment:
> Sure. Actually, for each 0<=N<=15 there's explicit ntuple(identity, Val{N}) method. The check for N<0 is in a generic method.

Ah, right! Wonseok's recent change. Thanks for reminding me. For `@code_llvm ntuple(identity, Val{16})` then :).